packages/remix-server-runtime/routeModules.ts

```diff
@@ -40,6 +40,9 @@ export type ResponseStubOperation = [
 export type ResponseStub = {
   status: number | undefined;
   headers: Headers;
+};
+
+export type ResponseStubImpl = ResponseStub & {
```

Comment:
This is what we use internally now - since it contains the symbol.  We cast to `ResponseStub` when handing off to loaders/actions